src/components/Search/SearchResultsList.tsx

```diff
@@ -4,12 +4,13 @@ import Box from '@mui/material/Box';
 import List from '@mui/material/List';
 import { connect } from 'react-redux';
 
+import useCheckRevision from '../../hooks/useCheckRevision';
 import type { Revision, State } from '../../types/state';
 import SearchResultsListItem from './SearchResultsListItem';
 
 function SearchResultsList(props: SearchResultsListProps) {
-  const { searchResults, handleChildClick } = props;
-
+  const { searchResults } = props;
+  const { checked, handleToggle } = useCheckRevision();
```

Comment:
Ah, also keeping this in a redux state would remove this handling from `SearchResultsList` component since it doesn't really need to know which revisions are checked. We could defer this logic directly to `SearchResultsListItem`.

Reply:
I've actually begun doing that in the PR to add revisions :ok_hand: